src/Game.ts

```diff
@@ -179,16 +162,16 @@ export class Game {
 			default:
 				throw new Error(`There shouldn't be an extra card type: ${this.board.colors[guessIndex]}`);
 		}
-
-		this.broadcastUpdatedBoard();
 	}
 
 	switchActiveTeam(): void {
     this.currTeam = this.currTeam === Team.red ? Team.blue : Team.red;
 		this.turn = Turn.spy;
-		var spymasters = this.findSpymasters();
+		
 		Broadcaster.switchActiveTeam(this.agents, this.currTeam, this.turn);
-		Broadcaster.promptForClue(spymasters[this.currTeam]);
+		
+		let currTeamSpy: Spymaster[] = gu.getByTeam(gu.getSpymasters(this.agents), this.currTeam)
```

Comment:
I'd change this to `currTeamMaster` or `currTeamSpymaster`, as spy could be confused with the agents that are normal players.